### PR TITLE
Fix "serde" and "serde_codegen" documentation links in README.md.

### PR DESCRIPTION
**Pull request by azasypkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ Serde Rust Serialization Framework
 Serde is a powerful framework that enables serialization libraries to
 generically serialize Rust data structures without the overhead of runtime type
 information. In many situations, the handshake protocol between serializers and
-serializees can be completely optimized away, leaving Serde to perform roughly
+serializes can be completely optimized away, leaving Serde to perform roughly
```
Comment (dtolnay) [inline README.md]: This was correct before -- serializee means "thing being serialized." See https://en.wiktionary.org/wiki/-ee#Suffix

Comment (azasypkin) [inline README.md]: Ah, right, will revert it back.

Comment (azasypkin) [inline README.md]: Done

**Comment by azasypkin** [comment]
Looks like something wrong with coverage check - change in README.md couldn't actually decrease code coverage for 0.03%.

**Comment by dtolnay** [comment]
It probably has to do with the release of Rust 1.7 this week. This PR was built against 1.7, 1.8, 1.9 whereas the baseline is against 1.6, 1.7, 1.8.

**Comment by erickt** [comment]
Thanks! I ended up fixing this in a separate parallel PR though.

**Pull request closed without merging**